code/modules/projectiles/guns/ego_gun/he.dm

```diff
@@ -34,7 +34,7 @@
 	special = "Use in hand to turn on homing mode. This mode fires slower, but homes in on a random target within 15 metres.	\
 			WARNING: This feature is not accurate."
 	ammo_type =	/obj/item/ammo_casing/caseless/ego_galaxy
-	fire_delay = 6
+	fire_delay = 13
```

Comment:
What about the attack speed change when homing is disabled? Is it not in your repo?

Reply:
Fuuuuuuuuuuuck